easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'MakeCp'
+
+name = 'Block'
+version = '1.5.3-20210723'
```

Comment:
I think this source code is 1.1.1.

From `block.spin_adapted-serial --version`
```
Block 1.1.1
```

Whereas the 1.5.3 downloaded from https://github.com/pyscf/Block/tree/master/bin reports back as `1.5`.

I cannot get the 1.5.3 source code link from https://sanshar.github.io/Block/build.html to work to check in more detail.

Reply:
Good question, this easyconfig is already using the sources from https://github.com/pyscf/Block, which is the reference repo according to https://github.com/sanshar/Block/issues/59

Maybe the binaries were updated to v1.5.3 but the source code is still old. I asked the question upstream, see
https://github.com/sanshar/Block/issues/59#issuecomment-1342770661

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16711/commits/d14524f868b0a6bb0adc699bdff956aadafca829